### PR TITLE
Android/feature/topic page

### PR DESCRIPTION
**Pull request by fihlie**
In #389 : Topic is the keyword for Wikidata's semantic tag. Topic Page contains detailed information about Topic. Such as:

Header part which contains name and description information about Wikidata semantic tag.
Related Topics, related topic data retrieved from backend.
Related Paths, related path data retrieved from backend.

--- 
For this PR, all the related features for topic page is completed.
Connection established between topic page and tag container on the search result
Connection established between topic page and tag container on the home page
Connection established between topic page and tag container on the path page.

### CODE REVIEW

File: source/android/Renaissance/lib/http_services.dart [hunk 1]
```diff
@@ -255,6 +255,58 @@ class HttpService {
...
+      /*List<Milestonee> milestoness = [];
+      List<Topic> topicss = [];
+
+      (temp["milestones"]).map((tag) {
+        milestoness.add(Milestonee.fromJson(tag));
+      }).toList();
+
+      (temp["topics"]).map((tag) {
+        topicss.add(Topic.fromJson(tag));
+      }).toList();*/
```
Comment (Hazerank) [inline source/android/Renaissance/lib/http_services.dart:273]: You can get rid od commented code blocks here.

File: source/android/Renaissance/lib/http_services.dart [hunk 2]
```diff
@@ -255,6 +255,58 @@ class HttpService {
...
+      throw Exception(res.body);
+    }
+  }
+
+  Future<List<Tag>> getTopicList(String topic_id) async {
+    String url = baseUrl + '/topic/related-topic/$topic_id/';
+    Response res = await get(Uri.parse(url), headers: headers);
+    if (res.statusCode == 200) {
+      Iterable l = json.decode(res.body);
+      List<Tag> basicPaths =
```
Comment (Hazerank) [inline source/android/Renaissance/lib/http_services.dart:289]: You should follow correct naming. Here topic list is stored in an array called basicPaths.

**Review by Hazerank — CHANGES_REQUESTED** [review]
The implementation of topic page looks neat! Thanks for your effort here. After fixing little issues with your code, we can merge it with peace of mind.

File: source/android/Renaissance/lib/http_services.dart (hunk 2 above)
Comment (fihlie) [reply source/android/Renaissance/lib/http_services.dart]: Done

File: source/android/Renaissance/lib/http_services.dart (hunk 1 above)
Comment (fihlie) [reply source/android/Renaissance/lib/http_services.dart]: of*

Comment (fihlie) [reply source/android/Renaissance/lib/http_services.dart]: Done

**Review by kursatbakis — APPROVED** [review]
I'm afraid my changes and yours are going to be highly conflicted and we're going to break something when trying to solve the conflicts. God help us. Except that, well-written code.

**Review by Hazerank — APPROVED** [review]
Thanks for the fixes and updates man!

**Pull request merged**